spec/lib/video_info_spec.rb

```diff
@@ -24,6 +24,29 @@
       expect { VideoInfo.new(url, options) }.to raise_error(VideoInfo::UrlError)
     end
   end
+  
+  describe ".disable_providers" do
+    let(:youtube_url) { 'https://www.youtube.com/watch?v=mZqGqE0D0n4'}
+    let(:vimeo_url) { 'http://vimeo.com/86701482' }
+
+    it "does not attempt to use a provider marked as disabled" do
+      VideoInfo.disable_providers = %w[YouTube Vimeo]
+
+      expect { VideoInfo.new(youtube_url) }.to raise_error(VideoInfo::UrlError)
+      expect { VideoInfo.new(vimeo_url) }.to raise_error(VideoInfo::UrlError)
+
+      VideoInfo.disable_providers = []
+    end
+
+    it "is case insensitive" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.